packages/babel-preset-env/src/index.ts

```diff
@@ -354,23 +351,20 @@ option \`forceAllTransforms: true\` instead.
   const exclude = transformIncludesAndExcludes(optionsExclude);
 
   const compatData = getPluginList(shippedProposals, bugfixes);
-  const shouldSkipExportNamespaceFrom =
-    (modules === "auto" && api.caller?.(supportsExportNamespaceFrom)) ||
-    (modules === false &&
-      !isRequired("transform-export-namespace-from", transformTargets, {
```

Comment:
No need to check `isRequired` here, given that it will be also re-checked as for any other plugin if the transform is not forcefully included.